common/flatpak-utils-http.c

```diff
@@ -596,13 +601,23 @@ flatpak_http_session_free (FlatpakHttpSession* session)
 static void
 set_error_from_curl (GError **error,
                      const char *uri,
-                     CURLcode res)
+                     CURLcode res,
+                     GCancellable *cancellable)
```

Comment:
```suggestion
set_error_from_curl (GError       **error,
                     const char    *uri,
                     CURLcode       res,
                     GCancellable  *cancellable)
```

Reply:
And the suggestion did not apply on github for whatever reason